src/Universe.cpp

```diff
@@ -610,6 +648,24 @@ void Universe::removeCell(Cell* cell) {
 
 
 /**
+ * @brief
+ * @param
+ * @returns
+ */
+bool Universe::containsPoint(Point* point) {
+
+  /* Loop over all Cells */
+  std::map<int, Cell*>::iterator iter;
+  for (iter = _cells.begin(); iter != _cells.end(); ++iter) {
+    if (!iter->second->containsPoint(point))
+      return false;
+  }
```

Comment:
It seems to me like this should be the other way around - when looping over the cells in a universe check to see if any of them contain the point. If one does, then return true. If none are found to contain the point, return false.